pkg/ibmvpc/vpc.go

```diff
@@ -21,6 +21,24 @@ func getNodeName(name, addr string) string {
 	return fmt.Sprintf("%s[%s]", name, addr)
 }
 
+type Region struct {
+	name string
+}
+
+var (
+	regionToStructMap = make(map[string]*Region)
```

Comment:
Why is this cache defined globally? I think it can defined in the function `VPCConfigsFromResources`